code/datums/mutations.dm

```diff
@@ -530,7 +530,17 @@
 
 /datum/mutation/human/swedish/say_mod(message)
 	if(message)
-		message = replacetext(message,"w","v")
+		message = replacetextEx(message,"W","V")
+		message = replacetextEx(message,"w","v")
+		message = replacetextEx(message,"J","Y")
+		message = replacetextEx(message,"j","y")
+		message = replacetextEx(message,"A",pick("�","�","�","A"))
+		message = replacetextEx(message,"a",pick("�","�","�","a"))
```

Comment:
tested it and it works fine for me, I don't play on Russian servers or have any special versions of byond